Package.swift

```diff
@@ -36,7 +36,6 @@ let package = Package(
         .package(url: "https://github.com/SwiftPackageIndex/SemanticVersion.git", from: "0.3.0"),
         .package(url: "https://github.com/SwiftPackageIndex/ShellOut.git", from: "3.1.4"),
         .package(url: "https://github.com/swiftlang/swift-package-manager.git", branch: "release/5.10"),
-        .package(url: "https://github.com/dankinsoid/VaporToOpenAPI.git", from: "4.4.4"),
```

Comment:
VaporToOpenAPI is not thread-safe. It causes crashes when running tests in parallel. Rather than create `openapi.json` dynamically on the fly from the routes, we should switch to using the [swift-openapi-generator](https://swiftpackageindex.com/apple/swift-openapi-generator).

Since the OpenAPI spec isn't a critical piece I've simply removed it for now.